setup_claude_desktop.py

```
@@ -0,0 +1,217 @@
+            venv_dir = current_dir / "venv"
+            if venv_dir.exists():
+                if platform.system() == "Windows":
+                    python_exe = str(venv_dir / "Scripts" / "python.exe")
+                else:
+                    python_exe = str(venv_dir / "bin" / "python")
+            else:
+                python_exe = sys.executable
+
+        # Try to import the required modules
+        result = subprocess.run(
+            [python_exe, "-c", "from asr_got.core import ASRGoTProcessor; print('✓ ASR-GoT modules imported successfully')"],
```

Pylint (3.3.7)

[convention] Line too long (125/100)

(C0301)